main.py

```diff
@@ -1,10 +1,45 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for key, value in players.items():
+        race = value.get("race")
+        if race:
+            race, bool_ = Race.objects.get_or_create(
+                name=race.get("name"),
```

Comment:
```bool_``` is not used anywhere in the code, you can hide it by ```_```

Reply:
I agree, fix that. Also avoid such naming, it is a bad practice